core/server/api/v2/db.js

```diff
@@ -23,6 +22,7 @@ module.exports = {
             }
         },
         query(frame) {
+            Object.assign(frame.options, {include: frame.options.withRelated});
```

Comment:
That is much better 👍 , even though it's hard to understand. I know it sucks that we have two uses cases for `include`, but from user perspective it makes sense i think.

Another option is to make serializers configureable in the controller e.g.

```
serializers: {
  input: {
    all: {
       toModelOptions: false
    }
  }
}
```
You can also access `frame.original.query.include`, BUT that value is not validated/serialized ☹️

Reply:
> BUT that value is not validated/serialized
Yeah, that's exactly why didn't go that way :+1:

Reply:
Let's just use this approach for now 👍 Maybe add a comment to the code base?

Reply:
:+1: comment added :)